serverlessworkflow/modules/ROOT/pages/service-orchestration/orchestration-of-openapi-based-services.adoc

```diff
@@ -29,9 +29,9 @@ This document describes how to call REST services using an link:{open_api_spec_u
 .Prerequisites
 
 include::../../pages/_common-content/getting-started-requirement.adoc[]
-* You have installed the Kogito tooling.
+* You have installed the tooling.
 +
-For more information about the tooling, see {getting-familiar-with-our-tooling}[Using Serverless Workflow tooling].
+For more information about the tooling, see {getting-familiar-with-our-tooling}[Getting familiar with Serverless Workflow tooling].
```

Comment:
I think here we should use the product name

Reply:
Done